exercises/concept/valentines-day/test/Tests.hs

```diff
@@ -0,0 +1,38 @@
+import Test.Hspec (describe, hspec, it)
+import ValentinesDay (Activity (..), Approval (..), Cuisine (..), Genre (..), rateActivity)
+
+main :: IO ()
+main = hspec $
+  describe "ValentinesDay" $ do
+    it "chill rated no" $
+      case rateActivity Chill of
+        No -> True
+        _ -> False
```

Comment:
Surprising. It correctly rejects incorrect implementations, but has the disadvantage of not showing the correct vs incorrect result. I wish there were a way to do that.